File: find all numbers disappeared.py
class Solution:
    def findDisappearedNumbers(self, nums: List[int]) -> List[int]:
        n=len(nums)
        t=[0]*(n+1)
        x=[]
        t[0]=1
        for i in nums:
            t[i]=1
        l=len(t)
        for i in range(l):
            if t[i]==0:
                x+=[i]
        return x

File: find all numbers disappeared.cpp
class Solution {
public:
    vector<int> findDisappearedNumbers(vector<int>& nums) {
        int l=nums.size();
        vector<int>t(l+1,0);
        vector<int>x;
        t[0]=1;
        for(int i=0;i<l;i++)
            t[nums[i]]=1;
        for(int j=0;j<l+1;j++)
            if(t[j]==0)
                x.push_back(j);
    return x;
    }
};
